### PR TITLE
Show the default processor list configuration in getting-started

### PR DESCRIPTION
**Pull request by AndydeCleyre**
This facilitates making small and informed changes to the default processor list before delving into deeper docs or code. Mentioned at the end of #173, though it is not addressed to that bug.

### CODE REVIEW

**Comment by hynek** [comment]
I agree that this would be a good thing, but need some way to ensure, that it doesn't go out of date. 🤔

Also maybe mention _all_ settings?

**Comment by AndydeCleyre** [comment]
Sure, I can try to add all the default settings. As for ensuring accuracy, maybe we turn the snippet into a doctest one which creates two loggers and asserts some kind of configuration equivalence?

**Comment by AndydeCleyre** [comment]
So instead, we could do something like below. Three issues to consider, though:

1. It's verbose `¯\_(ツ)_/¯`
2. This use of `ConsoleRenderer` fails the doctest because `colorama` is not in the `dev`, `tests`, `docs`, or standard requirements. Should I specify `colors=False`, or test for presence of `colorama`, or should it be included in `tests` or `dev` requirements, or something else?
3. Something's funky with `TimeStamper`, such that while two instances appear to have the same `type`, those `type`s equate neither via `is` nor `==`
4 (bonus consideration!). My local tests fail the `examples.rst` batch, indicating I may not be testing in a proper environment somehow

```rst
Below, we explicitly configure ``structlog`` to use its defaults.
The result is the same as not specifying any configuration:

.. doctest::

   >>> import structlog
   >>> default_cfg = structlog.get_config()  # save defaults for comparison
   >>> structlog.configure(
   ...     processors=[
   ...         structlog.processors.StackInfoRenderer(),
   ...         structlog.processors.format_exc_info,
   ...         structlog.processors.TimeStamper(),
   ...         structlog.dev.ConsoleRenderer()
   ...     ],
   ...     wrapper_class=structlog.BoundLogger,
   ...     context_class=dict,
   ...     logger_factory=structlog.PrintLoggerFactory(),
   ...     cache_logger_on_first_use=False
   ... )
   >>> # confirm that the explicit configuration is equivalent to the default configuration:
   >>> import types
   >>> explicit_cfg = structlog.get_config()
   >>> for prop in ('wrapper_class', 'context_class', 'cache_logger_on_first_use'):
   ...     assert default_cfg[prop] is explicit_cfg[prop]
   >>> assert type(default_cfg['logger_factory']) is type(explicit_cfg['logger_factory'])
   >>> assert len(default_cfg['processors']) == len(explicit_cfg['processors'])
   >>> for dproc, eproc in zip(default_cfg['processors'], explicit_cfg['processors']):
   ...     if isinstance(dproc, types.FunctionType):
   ...         assert dproc is eproc
   ...     else:
   ...         assert type(dproc) is type(eproc)
```

**Comment by AndydeCleyre** [comment]
@hynek Could you comment on the previous post?

**Comment by hynek** [comment]
I'm gonna say let's go a clean and easy way:

1. all of the config
2. put a comment in _config.py that reminds that getting-started.rst needs to be updated whenever the defaults change
3. profit

File: docs/getting-started.rst [hunk 1]
```diff
@@ -42,12 +42,18 @@ It should be noted that even in most complex logging setups the example would st
...
-   ])
+   structlog.configure(
+       processors=[
+           structlog.processors.StackInfoRenderer(),
+           structlog.processors.format_exc_info,
+           structlog.processors.TimeStamper(),
+           structlog.dev.ConsoleRenderer()
+       ],
+       wrapper_class=structlog.BoundLogger,
+       context_class=dict,
```
Comment (hynek) [inline docs/getting-started.rst]: could you add a comment here explaining that it's if the runtime has ordered dicts and OrderedDict otherwise?

Comment (AndydeCleyre) [reply docs/getting-started.rst]: Of course, thanks.

**Comment by hynek** [comment]
Thanks!

**Pull request merged**